### PR TITLE
remove sandboxctl "skip-workspace-prompt" argument

### PR DESCRIPTION
**Pull request by daniel-sanche**
We previously had the `--skip-workspace-prompt` to allow our automation to work without interaction. Now that the workspace set-up requirement is gone, we can remove the argument

### CODE REVIEW

File: .github/workflows/e2e-release.yml [hunk 1]
```diff
@@ -53,13 +53,10 @@ jobs:
         set -x
         docker pull ${{ steps.website_variables.outputs.cloudshell_image }}
         # run install script
-        # ISTIO_VERSION is fix for v0.5.0. Can be removed after next release
         docker run --rm \
           -e project_id=${{ env.PROJECT_ID }} \
-          -e skip_workspace_prompt=1 \
           -e release_repo=${{ steps.website_variables.outputs.repo }} \
           -e release_branch=${{ steps.website_variables.outputs.branch }} \
-          -e ISTIO_VERSION=1.7.1 \
```
Comment (simonz130) [inline .github/workflows/e2e-release.yml:62]: how is this related to your PR?

Comment (daniel-sanche) [reply .github/workflows/e2e-release.yml:62]: It's semi-relevant since it's another abandoned flag I found as I was working. But we don't have to address this here

**Review by simonz130 — APPROVED** [review]
